app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/HistoryTransactionDetailsBottomSheetFragment.kt

```diff
@@ -0,0 +1,229 @@
+package org.p2p.wallet.history.ui.detailsbottomsheet
+
+import androidx.annotation.StringRes
+import androidx.core.view.isGone
+import androidx.core.view.isVisible
+import androidx.fragment.app.FragmentManager
+import android.view.View
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.snackbar.Snackbar
+import org.koin.android.ext.android.inject
+import org.koin.core.parameter.parametersOf
+import org.p2p.uikit.glide.GlideManager
+import org.p2p.uikit.utils.getColor
+import org.p2p.uikit.utils.setTextColorRes
+import org.p2p.wallet.R
+import org.p2p.wallet.common.mvp.BaseMvpBottomSheet
+import org.p2p.wallet.databinding.DialogHistoryTransactionDetailsBinding
+import org.p2p.wallet.history.model.TransactionDetailsLaunchState
+import org.p2p.wallet.transaction.model.TransactionStatus
+import org.p2p.wallet.utils.Base58String
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.copyToClipBoard
+import org.p2p.wallet.utils.setStatus
+import org.p2p.wallet.utils.showInfoDialog
+import org.p2p.wallet.utils.showUrlInCustomTabs
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+import org.p2p.wallet.utils.withTextOrGone
+
+private const val EXTRA_STATE = "EXTRA_STATE"
+
+class HistoryTransactionDetailsBottomSheetFragment :
```

Comment:
i would even consider this fragment as a tech-debt and migrate it to recycler_view